external/kudu/src/main/scala/org/apache/gearpump/external/kudu/dsl/KuduDSLSink.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gearpump.external.kudu.dsl
+
+import scala.language.implicitConversions
+import org.apache.gearpump.cluster.UserConfig
+import org.apache.gearpump.external.kudu.KuduSink
+import org.apache.gearpump.streaming.dsl.scalaapi.Stream
+
+/** Create a Kudu DSL Sink */
+class KuduDSLSink[T](stream: Stream[T]) {
+
+  def writeToKudu(userConfig: UserConfig, table: String, parallelism: Int, description: String)
```

Comment:
Please try writing an example with this DSL sink

Reply:
KuduSinkDSL?

Reply:
can you give a hint on this? how can I call KuduDSLSink?

Reply:
Please checkout `KafkaReadWrite` example

Reply:
but kudu is sink, it doesn't work as a source.

Kafka can work as source and sink.

  def createAtMostOnceStream[T](
      app: StreamApp,
      topics: String,
      properties: Properties,
      parallelism: Int = 1,
      config: UserConfig = UserConfig.empty,
      description: String = "KafkaSource"
      ): Stream[T] = {
    app.source[T](new KafkaSource(topics, properties), parallelism, config, description)
  }
How would I convert above snippet into kudu one?

Reply:
Like `app.source(new Split).sink(new HBaseSink)`

Reply:
You meant to write a example like "app.source(new Split).sink(new HBaseSink)" for kudusinkdsl??

Reply:
Exactly